feature-group-definitions/sanitizer.yml

```
@@ -1 +1,2 @@
+name: "Sanitizer API"
```

Hmm, I don't think this needs "API". The API entrypoint is the `Sanitizer` constructor.

```suggestion
name: "Sanitizer"
```